.github/workflows/tests.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v2
       with:
-        go-version: '^1.16.4'
+        go-version: '~1.16.4'
```

Comment:
FYI: `'^1.16.4'` was matching with the latest go version 1.17.x - `^` can use any minor or patch version bigger than the specified one. We changed it to `~` to only match with any bigger patch version.